WordPress/gutenberg
# ComboboxControl: Update reset button size

In preparation for #65751

## What?

Updates the reset button size for ComboboxControl to align with the current sizing scheme.

## Testing Instructions

See the Storybook story for ComboboxControl and enable the `allowReset` prop. (And the `__next40pxDefaultSize` prop, which will be enabled in the story as part of #65751)

## Screenshots or screencast 

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/30a77fdc-3829-4ad1-bf10-45776f3193c1" alt="ComboboxControl reset button, before" width="270">|<img src="https://github.com/user-attachments/assets/61a7382f-2aa8-42bb-886c-479ad59b3767" alt="ComboboxControl reset button, after" width="270">|
